pyap_beauhurst/api.py

```diff
@@ -7,11 +7,16 @@
     :copyright: (c) 2015 by Vladimir Goncharov.
     :license: MIT, see LICENSE for more details.
 """
+from typing import Any, List, Optional
+
+import pyap_beauhurst.address
```

Comment:
maybe importing just `Address` might make the typing easier to read
`from pyap_beauhurst.address import Address`?